app/models/components/course/duplicator_component_host.rb

```diff
@@ -0,0 +1,15 @@
+class Duplicator::ModelComponentHost
```

Comment:
oops wasn't clear. I meant that mixing it into course model host would be the same as putting it straight into the course model host file, except the mixin method would have 2 files instead.